### PR TITLE
Resolving test cases, merge issues and system errors

### PR DESCRIPTION
**Pull request by AsthaBhalodiya**

### CODE REVIEW

File: create_pyenv.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```
Comment (abhira0) [inline create_pyenv.sh]: remove this file

File: tream release2024.2 [hunk 2]
```diff
@@ -0,0 +1,14 @@
+* [32mab-changes[m
```
Comment (abhira0) [inline tream release2024.2]: please remove this

File: pytest.ini [hunk 3]
```diff
@@ -2,3 +2,5 @@
 filterwarnings =
     ignore::DeprecationWarning
 addopts = --cov=api --cov-report=term-missing
+markers =
```
Comment (abhira0) [inline pytest.ini]: why do we need this?

File: api/tests/test_analytics.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
+# import pytest
```
Comment (abhira0) [inline api/tests/test_analytics.py]: uncomment this part

File: README.md [hunk 5]
```diff
@@ -42,6 +47,4 @@ A REST API application for managing expenses. Build your own automation, be it t
 [![GitHub Actions](https://img.shields.io/badge/github%20actions-%232671E5.svg?logo=githubactions&logoColor=white)](https://github.com/features/actions)
 [![Linux](https://img.shields.io/badge/Linux-FCC624?logo=linux&logoColor=black)](https://www.linux.org/)
 [![Visual Studio Code](https://img.shields.io/badge/Visual%20Studio%20Code-0078d7.svg?logo=visual-studio-code&logoColor=white)](https://code.visualstudio.com/)
-[![Zoom](https://img.shields.io/badge/Zoom-2D8CFF?logo=zoom&logoColor=white)](https://www.zoom.com/)
-[![DigitalOcean](https://img.shields.io/badge/DigitalOcean-%230167ff.svg?logo=digitalOcean&logoColor=white)]([#](https://www.digitalocean.com/))
-[![ChatGPT](https://img.shields.io/badge/ChatGPT-74aa9c?logo=openai&logoColor=white)](https://chatgpt.com/)
+
```
Comment (abhira0) [inline README.md]: add whats removed

File: Makefile [hunk 6]
```diff
@@ -1,3 +1,8 @@
+# Define variables
```
Comment (abhira0) [inline Makefile]: remove all the changes from this file

**Review by abhira0 — CHANGES_REQUESTED** [review]

File: pytest.ini (hunk 3 above)
Comment (AsthaBhalodiya) [reply pytest.ini]: required in some test cases

**Review by abhira0 — APPROVED** [review]

**Pull request merged**